release/release_logs/1.7.0/benchmarks/many_actors.txt

```diff
@@ -0,0 +1 @@
+Success! Started 100 actors in 2.97153377532959s. (33.65265467625668 actors/s)
```

Comment:
in 1.6.0 release logs, it is `Success! Started 10000 actors in 30.97493314743042s. (322.84169758828256 actors/s)
`, here we only started 100, is this expected?

Reply:
Hmm this is weird. @wuisawesome did we change anything from this test recently? Also @iycheng who worked on the actor scalability

Reply:
That sounds suspiciously like the smoke test￼

Reply:
I think I did it wrong here. I should get results from weekly release, not core nightly.